Podcast/Extensions.swift

```diff
@@ -43,3 +44,18 @@ extension UIFont {
     @nonobjc static let discoverTableViewCellDefaultFontAttributes = [NSForegroundColorAttributeName: UIColor.podcastGrayLight, NSFontAttributeName: UIFont(name: ".SFUIText-Medium", size: 10.0)!]
 
 }
+
+extension CMTime {
+    var durationText:String {
+        let totalSeconds = Int(self.seconds)
+        let hours = totalSeconds / 3600
```

Comment:
I think we don't really need the float/double precision for calculating the display time. The durationText is just going to be used for displaying on the Player UI in the format `h:mm:ss` or `mm:ss` so we never need to calculate this time in terms of fractions of a second.

Maybe that wasn't clear in the CMTime extension, so I just added a line of documentation of it to clarify.